Fix: Macro keys can't be used when USB MaxPacketSize = 64byte

ChnMasterOG opened the pull request:
Hello, 
I found macro keys are unavailable when my keyboard was used MaxPacketSize = 64byte protocol.
![image](https://github.com/user-attachments/assets/a39b56ce-e30d-4277-aaf8-e83fa59fb166)
![image](https://github.com/user-attachments/assets/8973eeb2-6378-45cb-a65d-8a37fcf96eae)
When I use 64B protocol, the remain 32B data will be judged as macro key raw data by the current codes.

ChnMasterOG committed:
Fix: Macro keys can't be used when USB MaxPacketSize = 64byte

ChnMasterOG committed:
change keyboards build path

ChnMasterOG committed:
Revert "change keyboards build path"

This reverts commit 2ba146197ee7e6fecf050ef6dbacbf85503ec60a.

Azarattum commented:
Good call. I was just going to report this. My keyboard also uses 64 byte reports. Hope this will get merged soon

ChnMasterOG committed:
update vite.config.ts

ChnMasterOG committed:
Revert "update vite.config.ts"

This reverts commit e4109bf96460004b4051bac47e31479cc19f547f.

lyonscw committed:
Align MAG60 VIA matrix

lyonscw committed:
更新 package-lock.json，修正应用名称并添加多个 esbuild 相关模块
优化宏录制器组件中的数组处理逻辑
修复 HID 设备请求逻辑，确保返回多个设备
在重新加载连接设备时添加错误日志处理
更新 HID 键盘工具函数以支持强制请求

Commit:
添加 CodeGraph 配置文件和忽略文件
